ethstaker_deposit/utils/validation.py

```diff
@@ -427,9 +427,46 @@ def validate_bls_to_execution_change_keystore(validator_index: str,
     )
 
     domain = compute_bls_to_execution_change_keystore_domain(
-        fork_version=chain_settings.GENESIS_FORK_VERSION,
-        genesis_validators_root=chain_settings.GENESIS_VALIDATORS_ROOT
+        fork_version=chain_setting.GENESIS_FORK_VERSION,
+        genesis_validators_root=chain_setting.GENESIS_VALIDATORS_ROOT
     )
 
     signing_root = compute_signing_root(message, domain)
     return bls.Verify(bls_pubkey, signing_root, bls_signature)
+
+
+#
+# Devnet Chain Setting Validation
+#
+
+def validate_devnet_chain_setting(ctx: click.Context, param: Any, value: Optional[str]) -> Optional[str]:
+    if value is None:
+        return None
+
+    trimmed_value = value[:400]
+
+    if validate_devnet_chain_setting_json(trimmed_value):
+        return trimmed_value
+    else:
+        raise ValidationError(load_text(['err_invalid_devnet_chain_setting']) + '\n')
+
+
+def validate_devnet_chain_setting_json(json_value: str) -> bool:
+    try:
+        devnet_chain_setting_dict = json.loads(json_value)
+
+        if not isinstance(devnet_chain_setting_dict, dict):
+            raise ValidationError(load_text(['err_devnet_chain_setting_not_object']) + '\n')
+
+        required_keys = ('network_name', 'genesis_fork_version', 'exit_fork_version')
+
+        all_keys = all(key in devnet_chain_setting_dict for key in required_keys)
+
+        if not all_keys:
+            raise ValidationError(load_text(['err_devnet_chain_setting_missing_keys']) + '\n')
+
+        return all_keys
```

Comment:
Since the JSON is very specific can we add two more checks here:
- Verify the number of keys are 3 or 4 to make it clear additional parameters are not being used if added for whatever reason
- Verify, if there is a fourth key, it is exactly `genesis_validator_root`